apps/music/js/ui/views/player_view.js

```diff
@@ -1095,7 +1095,8 @@ var PlayerView = {
         // If we're within 1 second of the end of the song, register
         // a timeout to skip to the next song one second after the song ends
         if (this.audio.currentTime >= this.audio.duration - 1 &&
-            this.endedTimer == null) {
+            this.endedTimer == null &&
+            this.playStatus !== PLAYSTATUS_PAUSED) {
           var timeToNext = (this.audio.duration - this.audio.currentTime + 1);
           this.endedTimer = setTimeout(function() {
```

Comment:
Fast seeking is broken because it changes the status away from PAUSED. Not sure what we expect, but to me if we are paused, fast seeking should not resume playback.